helm-charts/konk-service/values.yaml

```diff
@@ -53,6 +53,7 @@ ingress:
 konk:
   # The value below should be overriden by the konk-name
   name: ""
+  namespace: ""
```

Comment:
DEMO
Taken from chart, unless specified
```
helm template --set ingress.enabled=true --api-versions cert-manager.io/v1alpha3 helm-charts/konk-service | grep proxy-ssl-secret
    nginx.ingress.kubernetes.io/proxy-ssl-secret: default/-ingress-client
```

```
helm template --set konk.namespace=baz --set ingress.enabled=true --api-versions cert-manager.io/v1alpha3 helm-charts/konk-service | grep proxy-ssl-secret
    nginx.ingress.kubernetes.io/proxy-ssl-secret: baz/-ingress-client
```

Reply:
This is fine. It was implicitly `null` before.